src/main/scala/fr/acinq/bitcoin/Psbt.scala

```diff
@@ -80,12 +242,16 @@ object Psbt {
                                   sighashType: Option[Int],
                                   partialSigs: Map[PublicKey, ByteVector],
                                   derivationPaths: Map[PublicKey, KeyPathWithMaster],
-                                  redeemScript: Option[List[ScriptElt]],
-                                  witnessScript: Option[List[ScriptElt]],
-                                  scriptSig: Option[List[ScriptElt]],
+                                  redeemScript: Option[Seq[ScriptElt]],
+                                  witnessScript: Option[Seq[ScriptElt]],
+                                  scriptSig: Option[Seq[ScriptElt]],
                                   scriptWitness: Option[ScriptWitness],
                                   unknown: Seq[DataEntry]) extends DataMap
 
+  object PartiallySignedInput {
+    def empty: PartiallySignedInput = PartiallySignedInput(None, None, None, Map.empty, Map.empty, None, None, None, None, Nil)
```

Comment:
Nit: why  a `def` and not a `val` ?

Reply:
No reason really, updating that.

Reply:
Done in https://github.com/ACINQ/bitcoin-lib/pull/43/commits/79ecd0f68e12a909a485f318fcac1700fb6a8db6